core/src/test/resources/HistoryServerExpectations/completed_app_list_json_expectation.json

```diff
@@ -22,6 +23,7 @@
     "duration" : 101795,
     "sparkUser" : "jose",
     "completed" : true,
+    "appSparkVersion" : "",
```

Comment:
aah ok the UI will work because it is replaying but the rest end point would break as it is not allowing it to pass through, thanks @vanzin